randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -344,10 +349,14 @@ Extra - asset_id: 10
 > Door to Landing Site; Heals? False; Default Node
   * Layers: default
   * Normal Door to Landing Site/Door to Crateria Power Bomb Room
+  > Pickup (Power Bomb Expansion)
+      Trivial
 
 > Pickup (Power Bomb Expansion); Heals? False
   * Layers: default
   * Pickup 0; Category? Minor
+  > Door to Landing Site
+      Trivial
```

Comment:
See comment above

Reply:
I checked, and dragons can be killed with power beam. Nothing more needed here.